### PR TITLE
Fix layout & accessibility issues

### PR DESCRIPTION
**Pull request by LouisBorleeAdevinta**
### Layout
UIKit:
- Spacing between toggle and text not recalculated properly 
- View extension from static equalTo anchors to greaterThanOrEqualTo (or lessThanOrEqualTo for bottom)

UIKit & SwiftUI:
- Removed title & supplementary label (see [FormField](https://github.com/adevinta/spark-ios-component-form-field))

### Accessibility
UIKit:
- Added button trait (removes native handling of isSelected & isEnabled)
- Change isSelected management to handle isSelected trait manually
- Added isEnabled trait manually
- Make the radio button be part of the accessibility (was just the label)

SwiftUI:
- Added .button trait
- Managed .isSelected trait
- Updated accessibilityValue & accessibilityLabel

### CODE REVIEW

**Review by robergro — DISMISSED** [review]

File: Tests/SnapshotTests/.__snapshots__/RadioButtonUIViewSnapshotTests/Test5-main-horizontal-textLeading-withTapArea-defaultContainer-light-extraExtraExtraLarge.1.png
Comment (robergro) [inline Tests/SnapshotTests/.__snapshots__/RadioButtonUIViewSnapshotTests/Test5-main-horizontal-textLeading-withTapArea-defaultContainer-light-extraExtraExtraLarge.1.png:1]: Th green background is OK ?

File: Tests/SnapshotTests/.__snapshots__/RadioButtonUIViewSnapshotTests/Test1-accent-vertical-textTrailing-withoutTapArea-defaultContainer-light-medium.1.png
Comment (robergro) [inline Tests/SnapshotTests/.__snapshots__/RadioButtonUIViewSnapshotTests/Test1-accent-vertical-textTrailing-withoutTapArea-defaultContainer-light-medium.1.png:1]: The 4th radio seems to be focus, it is normal ..?

Comment (LouisBorleeAdevinta) [reply Tests/SnapshotTests/.__snapshots__/RadioButtonUIViewSnapshotTests/Test1-accent-vertical-textTrailing-withoutTapArea-defaultContainer-light-medium.1.png:1]: Yes, goal is to test all states of 
- isSelected
- isEnabled
- isHighlighted

Shouldn't happen all that much but it doesn't cost much to test either

File: Tests/SnapshotTests/.__snapshots__/RadioButtonUIViewSnapshotTests/Test5-main-horizontal-textLeading-withTapArea-defaultContainer-light-extraExtraExtraLarge.1.png
Comment (LouisBorleeAdevinta) [reply Tests/SnapshotTests/.__snapshots__/RadioButtonUIViewSnapshotTests/Test5-main-horizontal-textLeading-withTapArea-defaultContainer-light-extraExtraExtraLarge.1.png:1]: Yes, it's a way to test the tap area which we've had issue with in the past

**Review by robergro — APPROVED** [review]

**Pull request merged**